powerlevel9k.zsh-theme

```diff
@@ -337,6 +337,55 @@ prompt_background_jobs() {
   fi
 }
 
+# Segment to indicate hdd available level.
+set_default POWERLEVEL9K_HDD_USAGE_ONLY_WARNING false
+set_default POWERLEVEL9K_HDD_USAGE_WARNING_LEVEL 10
+set_default POWERLEVEL9K_HDD_USAGE_CRITICAL_LEVEL 5
+prompt_hdd_usage() {
```

Comment:
So, actually, this code does implement usage. It's using `df`, which by default will give you usage percent. It also gives you "available blocks", but that's not what the math is based on. Column 5 in the below: 
![usage](https://cloud.githubusercontent.com/assets/569571/22171212/559703aa-df57-11e6-8b9f-1ad7ae59d4d9.png)